hwpf/fapi2/src/plat/plat_utils.C

```diff
@@ -151,39 +151,9 @@ std::string plat_HwCalloutEnum_tostring(HwCallouts::HwCallout hwcallout)
 
     switch(hwcallout)
     {
-        case HwCallouts::HwCallout::TOD_CLOCK:
```

Comment:
Just a minor comment, Please align the commit message.

Reply:
Done